package-lock.json

```diff
@@ -3159,19 +3172,66 @@
         "file-uri-to-path": "1.0.0"
       }
     },
+    "bip-schnorr": {
```

Comment:
This is a `bitcore-lib` dependency, was upgraded to `0.6.4` in a version greater than `8.25.10`